pkg/apis/devconsole/v1alpha1/component_types.go

```diff
@@ -4,24 +4,24 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// ComponentSpec defines the desired state of Component
+// ComponentSpec defines the desired state of Component.
 // +k8s:openapi-gen=true
 type ComponentSpec struct {
 	// Container image use to build (nodejs, golang etc..)
 	BuildType string `json:"buildType"`
-	// Codebase is the source code of your component. Atm only public remote URL are supported.
-	Codebase string `json:"codebase"`
+	// GitSourceRef is the source code of your component. Currently, only public remote URLs are supported.
+	GitSourceRef string `json:"codebase"`
```

Comment:
You need to change the json tag too.

Reply:
https://github.com/redhat-developer/devconsole-api/pull/12

Reply:
Thanks a lot, I missed. :(